docs/odh/oci-model-storage.md

```diff
@@ -8,15 +8,227 @@ which also explains how to deploy models from OCI images.
 
 This page offers a guide similar to the upstream project documentation, but
 focusing on the OpenDataHub and OpenShift characteristics. To demonstrate
-how to create an OCI container image, the publicly available [MobileNet v2-7
-model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
-is used. This model is in ONNX format.
+how to create and use OCI containers, two examples are provided:
+* The first example uses [IBM's Granite-3.0-2B-Instruct model](https://huggingface.co/ibm-granite/granite-3.0-2b-instruct) 
+  available in Hugging Face. This is a generative AI model.
+* The second example uses the [MobileNet v2-7 model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+  is used. This is a predictive AI model in ONNX format.
+
+## Creating and deploying an OCI image of IBM's Granite-3.0-2B-Instruct model
+
+IBM's Granite-3.0-2B-Instruct model is [available at Hugging Face](https://huggingface.co/ibm-granite/granite-3.0-2b-instruct).
+To create an OCI container image, the model needs to be downloaded and copied into
+the container. Once the OCI image is built and published in a registry, it can be
+deployed on the cluster.
+
+The ODH projects provides configurations for the vLLM model server, which
+supports running the Granite model. Thus, this guide will use this model server
+to demonstrate how to deploy the Granite model stored in an OCI image.
+
+### Storing the Granite model in an OCI image
+
+To download the Granite model, the [`huggingface-cli download` command](https://huggingface.co/docs/huggingface_hub/guides/cli#huggingface-cli-download) can be used.
+The process is as follows:
+* Install the huggingface CLI
+* Use huggingface CLI to download the model
+* Create the final OCI using the downloaded model
+
+It is possible to run this process as part of a multi-stage container build. Create a file named
```

Comment:
By saying "it is possible", this looks like an alternative method to the process described above.
I would says "This process is implemented in the following multi-stage container build. Create a file..."

Reply:
Sounds better. I changed it.